book/src/storage.md

```diff
@@ -5,23 +5,23 @@ configured](configuration.md) in `~/.config/aura/config.toml`.
 
 ## Where are packages built?
 
-By default, packages are built within `~/.cache/aura/builds/`. Were a package
-specified in the `chroot` list in config, then it will instead be built in a
-`chroot`.
```

Comment:
Here I am using the subjunctive mood in English, which I quite enjoy as it is a rare form. Please revert the changes to this paragraph entirely and then I can merge this. The corrections below are good as-is, thank you.